lib/active_model/serializer.rb

```diff
@@ -1,14 +1,14 @@
 require 'thread_safe'
+require_relative 'serializer/configuration'
+require_relative 'serializer/associations'
```

Comment:
I'd rather use the full require `active_model/serializer/configuration` without further discussion